hashintel/hash
# Remove `ReadonlyModeProvider` & rename `useReadonlyMode`

## 🌟 What is the purpose of this PR?

This PR removes `ReadonlyModeProvider` and renames `useReadonlyMode` to `useIsReadonlyMode`. 

## 🔗 Related links





- [Asana task](https://app.asana.com/0/1203363157432084/1203535632757828/f) _(internal)_
- [Related slack thread](https://hashintel.slack.com/archives/D03QLNUHRM1/p1670857741409529) _(internal)_

## ❓ How to test this?



1.  Checkout the branch / view the deployment
1.  Confirm that `readonly mode` works as expected by checking a hash page with `?readonly` query param
